ReportingServicesTools.psm1

```diff
@@ -1,2 +1,17 @@
+﻿<#
+    .NOTES
+    --------------------------------------------------------------------------------
+     Code generated by:  SAPIEN Technologies, Inc., PowerShell Studio 2017 v5.4.135
+     Generated on:       05.02.2017 14:43
+     Generated by:       Friedrich Weinmann
```

Comment:
This one is still active please remove it before the final commit

---
In reply to: [100653489](https://github.com/Microsoft/ReportingServicesTools/pull/12#discussion_r100653489) [](ancestors = 100653489)